desktop/src/main/java/bisq/desktop/primary/main/content/trade/overview/TradeOverviewGridView.java

```diff
@@ -0,0 +1,175 @@
+        basicInfo.setMaxWidth(420);
+        basicInfo.setMinHeight(80);
+        box.getChildren().add(basicInfo);
+
+        Region line = new Region();
+        line.getStyleClass().addAll("border-bottom");
+        box.getChildren().add(line);
+
+        GridPane paramsPane = new GridPane();
+        paramsPane.setPadding(new Insets(24));
+        paramsPane.setVgap(28);
+        paramsPane.add(
+                getParameterPane(Res.get("markets"), protocol.getMarkets()), 
+                0, 
+                0
+        );
+        
+        paramsPane.add(
+                getParameterPane(
+                        Res.get("trade.protocols.table.header.release"), 
+                        protocol.getReleaseDate()
+                ), 
+                1, 
+                0
+        );
+        
+        paramsPane.add(
+                getParameterPane(
+                        Res.get("trade.protocols.table.header.security"), 
+                        protocol.getSwapProtocolType().getSecurity().ordinal(), 
+                        protocol.getSecurityInfo()
+                ), 
+                0, 
+                1
+        );
+        
+        paramsPane.add(
+                getParameterPane(
+                        Res.get("trade.protocols.table.header.privacy"),
+                        protocol.getSwapProtocolType().getPrivacy().ordinal(),
+                        protocol.getPrivacyInfo()
+                ), 
+                1, 
+                1
+        );
+        
+        paramsPane.add(
+                getParameterPane(
+                        Res.get("trade.protocols.table.header.convenience"),
+                        protocol.getSwapProtocolType().getConvenience().ordinal(),
+                        protocol.getConvenienceInfo()
+                ), 
+                2, 
+                1
+        );
+
+        box.getChildren().add(paramsPane);
+
+        return box;
+    }
+    
+
+    private VBox getParameterPane(String title, String info) {
+        Label infoLabel = new Label(info);
+        infoLabel.getStyleClass().add("bisq-text-1");
+        return getParameterPane(title, infoLabel);
+    }
+    
+    private VBox getParameterPane(String title, int value, String tooltipText) {
+        return getParameterPane(title, getStarsNode(value, tooltipText));
+    }
+    
+    private VBox getParameterPane(String title, Node node) {
+        VBox box = new VBox(6);
+        GridPane.setHgrow(box, Priority.ALWAYS);
+        Label titleLabel = new Label(title.toUpperCase());
+        titleLabel.getStyleClass().add("bisq-text-5");
+        box.getChildren().addAll(titleLabel, node);
+        
+        return box;
+    }
+
+    private Node getStarsNode(int value, String tooltipText) {
+        final HBox hBox = new HBox();
+        hBox.setSpacing(5);
+        hBox.setAlignment(Pos.CENTER_LEFT);
+
+        for (int i = 0; i < 3; i++) {
+            Label label = Icons.getIcon(AwesomeIcon.STAR, "1.3em");
```

Comment:
I think its best here to make full inheritence for all the MCV classes. So there are 2 controllers extending a basecontroller, same for models and views. Its a bit boilerplate but keeps structures clean, otherwise shortterm shortcuts lead to more messy structure later and over time more stuff gets in and then the initial boilerplate pays off.

Reply:
Services should only be used for real domain data, and then they are in the relevant domains (e.g. social). For pure UI state we should keep things in the models.